oceans/datasets/datasets.py

```diff
@@ -67,6 +67,7 @@ def woa_subset(llcrnrlon=2.5, urcrnrlon=357.5, llcrnrlat=-87.5, urcrnrlat=87.5,
     >>> import numpy.ma as ma
     >>> import matplotlib.pyplot as plt
     >>> from oceans.colormaps import cm, get_color
+    >>> from oceans.datasets import map_limits, woa_subset
     >>> from mpl_toolkits.basemap import Basemap
```

Comment:
The failure is unrelated to this.  Like I suspected, it was a server issue that seems to be fixed now.  I can also run it locally without the changes and soon we'll know if travis-ci can run it too.

However, for completeness of the example, the changes are welcome.  Thanks.